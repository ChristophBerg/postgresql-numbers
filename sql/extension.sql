SET client_min_messages = warning;
CREATE EXTENSION IF NOT EXISTS numeral;
RESET client_min_messages;
