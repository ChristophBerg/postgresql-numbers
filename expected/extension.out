SET client_min_messages = warning;
CREATE EXTENSION IF NOT EXISTS numbers;
RESET client_min_messages;
